pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -145,16 +145,6 @@ in stdenv.mkDerivation rec {
       --replace "SYSTEMD_CGROUP_AGENT_PATH" "_SYSTEMD_CGROUP_AGENT_PATH"
   '';
 
-  patches = [
-    # https://github.com/systemd/systemd/pull/8580
-    (fetchpatch {
-      url = https://github.com/systemd/systemd/pull/8580.patch;
-      sha256 = "1yp07hlpgqq0h2y0qc3kasswzkycz6p8d56d695ck1qa2f5bdfgn";
-    })
-  ];
-
-  hardeningDisable = [ "stackprotector" ];
```

Comment:
@fpletz can you recall, why this was disabled? It seems to compile for me, but I am not sure, if it has any effects at runtime.

Reply:
Most probably it didn't build at the time, if I recall correctly it was systemd-boot that failed. I'm quite sure it was at build time but nonetheless it might be good to try the bootloader test. I'll give it a shot.

Reply:
I run nixos/tests/boot.nix earlier and also use systemd-boot on my own system.